src/templates/blog-post.js

```diff
@@ -166,12 +168,12 @@ class BlogPostTemplate extends React.Component {
                   marginTop: rhythm(-4 / 5),
                 }}
               >
-                {formatPostDate(post.frontmatter.date, lang)}
+                {`Created: ${formatPostDate(post.frontmatter.date, lang)}`}
                 {` • ${formatReadingTime(post.timeToRead)}`}
-                {post.frontmatter.authors &&
```

Comment:
Bring authors back ?

Reply:
yup, since now going with the above suggestion.